tests/errors_tests.rs

```diff
@@ -1,4 +1,9 @@
+use crate::assert_matches::assert_matches;
 use crate::common::setup_mock_for;
+use dnsimple::errors::DNSimpleError;
+use serde_json::json;
+#[macro_use]
+extern crate assert_matches;
```

Comment:
```suggestion
use crate::common::setup_mock_for;
use assert_matches::assert_matches;
use dnsimple::errors::DNSimpleError;
use serde_json::json;
```

This is the correct way of defining the crate. the extern crate is very old syntax